src/api/DataHandler.ts

```diff
@@ -1,11 +1,61 @@
 import fetch from 'node-fetch';
 
-const IP = "http://localhost:8081" //"80.167.223.178";
+const api = "localhost:3306" //"80.167.223.178";
 
 enum HttpRequestType {
     Get,
     Post
 }
+//@ts-ignore
+function Login(email: string, password: string){
+    return new Promise((resolve, rejects) => {
+        fetch(api, {
+            method: 'POST',
+            headers: {
+                Accept: 'application/json',
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify({
+                email: email,
+                password: password
+            }),
+        }).then(response => resolve(response)).catch(err => rejects(err));
+    });
+}
+//@ts-ignore
+function Register(username: string, email: string, password: string){
+    return new Promise((resolve, rejects) => {
+        fetch(api, {
+            method: 'POST',
+            headers: {
+                Accept: 'application/json',
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify({
+                email: email,
+                username: username,
+                password: password
+            }),
+        }).then(response => resolve(response)).catch(err => rejects(err));
+    });
+}
+//@ts-ignore
+function ForgotPassword(email: string){
+    return new Promise((resolve, rejects) => {
+        fetch(api, {
+            method: 'POST',
+            headers: {
+                Accept: 'application/json',
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify({
+                email: email,
+            }),
+        }).then(response => resolve(response)).catch(err => rejects(err));
+    });
+}
+
+
 
 async function fetchData(url: string, requestType: HttpRequestType, bodyData?: string | object) {
```

Comment:
Consistancy with naming conventions, capital letter for function name, as the previous.

Reply:
Fixed in recent commit